src/D2L.Security.OAuth2/Validation/Request/HttpRequestExtensions.net4x.cs

```diff
@@ -1,6 +1,4 @@
-#if !DNXCORE50
-
-using System.Web;
+﻿using System.Web;
 
 namespace D2L.Security.OAuth2.Validation.Request {

```

Comment:
System.Web.HttpRequest only exists in .NET Framework.

A bunch of this code supports that in addition to `HttpRequestMessage`, the modern/ASP.NET Core version. In .NET Core we will _only_ support that (note this is a `.net4x.cs` file, and there are a bunch more below.)